temporalcli/commandsmd/commands.md

```diff
@@ -635,6 +635,29 @@ temporal workflow execute
 Includes options set for [workflow start](#options-set-for-workflow-start).
 Includes options set for [payload input](#options-set-for-payload-input).
 
+### temporal workflow history: Interact directly with a workflow's event history.
+
+Workflow history commands use this syntax: `temporal workflow history COMMAND [ARGS]`.
+
+### temporal workflow history convert: Converts an event history between formats.
+
+```
+temporal workflow history convert \
+                --input-format json \
+		--input-file original.json \
+                --output-format json \
+		--output-file reserialized.json
+```
+
+Use the options listed below to change the command's behavior.
+
+#### Options
+
+* `--source-format` (string-enum) - Format of the input file.  Options: json, proto, pb, prototext, pbtxt.  Default: json.
+* `--source-file` (string) - Path to the input file.  Required.
+* `--target-format` (string-enum) - Format of the output file.  Options: json, proto, pb, prototext, pbtxt.  Default: json.
```

Comment:
There is only one history format for Temporal (happens to be proto JSON). However there is a future where workflow export which wraps as proto binary may have use, but I don't think it's needed now.

Reply:
I mean, fair.  I didn't have a specific use case in mind, I just thought maybe it would be nice to have a command to do that if needed.  Maybe the CLI isn't the right place for that, though, since it's meant for consumer use.